administration/src/bp-modules/applications/JsonFieldElemental.tsx

```diff
@@ -96,7 +97,7 @@ const JsonFieldElemental = ({
     case 'String':
       return (
         <p>
-          {t(getTranslationKey())}: {jsonField.value}
```

Comment:
:upside_down_face: it may have been better to add a space here rather than inside EmailLink. Because it looks like a part of the link currently
![image](https://github.com/user-attachments/assets/fe54587c-bba9-4c57-b5db-316fea5eca96)